CIPs/cip-granda_mento.md

```diff
@@ -0,0 +1,187 @@
+---
+cip: <to be assigned>
+title: Granda Mento
+author: Trevor Porter (@tkporter), Albert Wang (@albertclabs)
+discussions-to: https://forum.celo.org/t/discussion-on-granda-mento-enabling-larger-stablecoin-mints/966
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): <Ring 0, 1, 2, 3>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+license: Apache 2.0
+---
+
+## Simple Summary
+
+Granda Mento is a mechanism to facilitate large CELO <-> stable token (e.g. cXXX) exchanges that aren't suitable via Mento or OTC. A new contract is created that can exchange `CELO <-> stable token` for any stable token Governance explicitly enables. Exchanges via this contract must be approved by a multisig and can be vetoed by Governance.
```

Comment:
Like you mentioned earlier, the multisig approver was intended to alleviate fatigue for voters. Governance will be able to set the `approver` address. There was some pushback against having Governance being required to approve every exchange. There were concerns that it would require too much participation if we expected Granda Mento to exist for more than a few months / exchanges. As it became more clear that this is something that'll exist for a decent amount of time (especially for smaller total supply stablecoins like cEUR or those that may be added in the future), the multisig approach became more attractive. I believe as long as the process is transparent, community members can give their input, and governance has the ability to step in if the multisig doesn't execute the will of the community, there are very similar benefits to exchanges being approved by governance

Fwiw there's a (very small) precedent of governance voting for vetoes I read about [here](https://ournetwork.substack.com/p/our-network-deep-dive-2)

Reply:
100% agree. The multisig owners haven't been decided yet, but I think that can be a convo that's had when the CGP is proposed that will activate GrandaMento & actually set the multisig.